### PR TITLE
Replaced old 404 image with blank frank picture and text

### LINKED ISSUE #417: bad contrast on 404 page dark mode
### What happened?

image

### PR DESCRIPTION
**Pull request by Angel-Armendariz**
Basically, I did the thingy with the thongy and used the blank Frank picture that Mike gave me with a 404 error message above it to fix the issue. It works on both mobile and desktop after doing some magic wizard stuff. I think it works alright. I apologize to Mike in advance because there is no question mark above Frank's head like in the original photo but I think he will find it in his heart to forgive me.

### CODE REVIEW

**Comment by jaasonw** [comment]
image
maybe make this gap smaller?

**Comment by Angel-Armendariz** [comment]
> image maybe make this gap smaller?
made the gap smaller in new commit, Let me know how that looks or if the gap is still too big 
image

**Comment by jaasonw** [comment]
to reiterate what i said on discord


a. need text-align: center
b. font doesn't appear to be bolded on my phone (tested on chrome on android)

**Comment by jaasonw** [comment]
image

**Comment by Angel-Armendariz** [comment]
> image

image

File: src/routes/__error.svelte [hunk 1]
```diff
@@ -8,21 +8,33 @@
...
+    margin-bottom: 2em;
+    align-items: center;
+    text-align: center;
+  }
+  section em {
+    align-items: center;
+    text-align: center;
+    font-size: 50px;
+    margin-bottom: 0.5em;
+    margin-top: 2em;
```
Comment (jaasonw) [inline src/routes/__error.svelte:38]: maybe add more to the bottom too and i think we're gucci

**Review by jaasonw — DISMISSED** [review]

File: src/routes/__error.svelte (hunk 1 above)
Comment (jaasonw) [reply src/routes/__error.svelte:37]: add more to*

Comment (Angel-Armendariz) [reply src/routes/__error.svelte:38]: > add more to*

So you want me to increase the margin-bottom? Like 1 or 2em?

Comment (jaasonw) [reply src/routes/__error.svelte:38]: ye the picture lookin kinda cramped on the bottom too if u make the window small enough

Comment (jaasonw) [reply src/routes/__error.svelte:38]: image

Comment (Angel-Armendariz) [reply src/routes/__error.svelte:37]: image
How's this my boy

Comment (Angel-Armendariz) [reply src/routes/__error.svelte:38]: 2em too strong? Or 1 em?
image

Comment (Angel-Armendariz) [reply src/routes/__error.svelte:38]: Or do you prefer 3em my boy
image
, this is 3em

Comment (jaasonw) [reply src/routes/__error.svelte:38]: 3's probably fine

Comment (Angel-Armendariz) [reply src/routes/__error.svelte:38]: Pushing now, I await your new bug that you will find soon 
image

File: src/routes/__error.svelte [hunk 2]
```diff
@@ -8,34 +8,52 @@
...
   }
```
Comment (EthanThatOneKid) [inline src/routes/__error.svelte:41]: To make the `<em>` stand out like it does in the original 404 graphic, we could add a [text-shadow property](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow) to this CSS block: 

Mode | Dark | Light
---|---|---
Before text-shadow | image | image
After text-shadow | image | image

```suggestion
    text-shadow: 2px 2px var(--acm-blue);
  }
```

File: src/routes/__error.svelte [hunk 3]
```diff
@@ -8,34 +8,52 @@
...
-
-  section .mobile {
-    display: none;
+  section h1 {
+    margin-bottom: 2em;
+    align-items: center;
+    text-align: center;
+  }
+  section em {
```
Comment (EthanThatOneKid) [inline src/routes/__error.svelte]: To make the CSS easier to read, we add a blank line separating each CSS definition.
```suggestion
  }
  
  section em {
```

File: src/routes/__error.svelte [hunk 4]
```diff
@@ -8,34 +8,52 @@
 </svelte:head>
 
 <section>
-  <img src="/assets/png/404-h.png" class="wide" alt="404 - Page Not Found" />
-  <img src="/assets/png/404-v.png" class="mobile" alt="404 - Page Not Found" />
+  <em>404</em>
+  <h1>Frank can't find where you're going!</h1>
+  <img src="/assets/png/lost-frank.png" class="wide" alt="404 - Page Not Found" />
+  <img src="/assets/png/lost-frank.png" class="mobile" alt="404 - Page Not Found" />
```
Comment (EthanThatOneKid) [inline src/routes/__error.svelte]: I noticed that the `wide` and `mobile` class names are not in use, so you can go ahead and remove one of the images.
```suggestion
  <img src="/assets/png/lost-frank.png" alt="404 - Page Not Found" />
```

File: src/routes/__error.svelte [hunk 5]
```diff
@@ -8,34 +8,52 @@
...
     max-width: 1000px;
     height: auto;
+    margin-bottom: 3em;
+  }
+
+  @media (min-width: 900px) {
+    section .mobile {
+      display: none;
+    }
   }
```
Comment (EthanThatOneKid) [inline src/routes/__error.svelte]: Unless I misunderstand, the `wide` and `mobile` class names seem to not affect anything. If so, you can go ahead and remove these lines of CSS.
```suggestion
```

**Review by EthanThatOneKid — DISMISSED** [review]
Congrats on finishing up with finals; _phew_! Managed to take a look at your PR again and added some minor suggestions. Progress looks great so far! Thanks for contributing :)

File: src/routes/__error.svelte (hunk 2 above)
Comment (EthanThatOneKid) [reply src/routes/__error.svelte:41]: I used `--acm-blue` for this suggestion, but the original 404 graphic used `#92c4df`.

image

**Comment by Angel-Armendariz** [comment]
> Congrats on finishing up with finals; _phew_! Managed to take a look at your PR again and added some minor suggestions. Progress looks great so far! Thanks for contributing :)

Congrats to you as well, had a good time learning the ropes with this issue. Let me know if there are any other problems with it

File: src/routes/__error.svelte (hunk 5 above)
Comment (Angel-Armendariz) [reply src/routes/__error.svelte]: done

File: src/routes/__error.svelte (hunk 4 above)
Comment (Angel-Armendariz) [reply src/routes/__error.svelte]: done

File: src/routes/__error.svelte (hunk 3 above)
Comment (Angel-Armendariz) [reply src/routes/__error.svelte]: done

File: src/routes/__error.svelte (hunk 2 above)
Comment (EthanThatOneKid) [reply src/routes/__error.svelte:41]: > I used `--acm-blue` for this suggestion, but the original 404 graphic used `#92c4df`.
> 
> image

To better match the original 404 graphic, you are able to add some more weight, lighten the color, and increase the font size as needed (while keeping in mind how the element responds on mobile and wide screens).
```suggestion
    /* #92c4df is slightly lighter than var(--acm-blue) */
    /* see <https://etok.codes/acmcsuf.com/pull/458/files#r878929505> */
    text-shadow: 2px 2px #92c4df;
    font-weight: 600;
  }
```

Comment (Angel-Armendariz) [reply src/routes/__error.svelte:41]: Done

**Review by k4rni — APPROVED** [review]

**Pull request merged**